control-plane/tests/test_phase28_misp_enrichment_validation.py

```
@@ -0,0 +1,396 @@
+        service.persist_record(
+            replace(
+                promoted_case,
+                evidence_ids=(*promoted_case.evidence_ids, anchor_evidence.evidence_id),
+            )
+        )
+        return anchor_evidence
+
+    def test_attach_misp_context_persists_subordinate_attachment_with_provenance(
+        self,
+    ) -> None:
+        store, service, promoted_case, _, reviewed_at = self._build_in_scope_case(
```

Ruff (0.15.10)

[warning] Unpacked variable `store` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)